### ISSUE TITLE
Why all AP are zero??

### ISSUE THREAD

**Issue by AN-Ahmed** [traveller59/second.pytorch#450]
(pointpillars) ahmed@ahmed:~/ahmed/code/second.pytorch/second$ python3 ./pytorch/train.py evaluate --config_path=./configs/car.fhd.config --model_dir=/home/ahmed/code/second.pytorch/second/pretrained_models_v1.5/car_fhd/ --measure_time=True --batch_size=1
/home/ahmed/anaconda3/envs/pointpillars/lib/python3.7/site-packages/numba/cuda/envvars.py:17: NumbaWarning: 
Environment variables with the 'NUMBAPRO' prefix are deprecated and consequently ignored, found use of NUMBAPRO_CUDA_DRIVER=/usr/lib/x86_64-linux-gnu/libcuda.so.

For more information about alternatives visit: ('https://numba.pydata.org/numba-doc/latest/cuda/overview.html', '#cudatoolkit-lookup')
  warnings.warn(errors.NumbaWarning(msg))
/home/ahmed/anaconda3/envs/pointpillars/lib/python3.7/site-packages/numba/cuda/envvars.py:17: NumbaWarning: 
Environment variables with the 'NUMBAPRO' prefix are deprecated and consequently ignored, found use of NUMBAPRO_NVVM=/usr/local/cuda/nvvm/lib64/libnvvm.so.

For more information about alternatives visit: ('https://numba.pydata.org/numba-doc/latest/cuda/overview.html', '#cudatoolkit-lookup')
  warnings.warn(errors.NumbaWarning(msg))
/home/ahmed/anaconda3/envs/pointpillars/lib/python3.7/site-packages/numba/cuda/envvars.py:17: NumbaWarning: 
Environment variables with the 'NUMBAPRO' prefix are deprecated and consequently ignored, found use of NUMBAPRO_LIBDEVICE=/usr/local/cuda/nvvm/libdevice.

For more information about alternatives visit: ('https://numba.pydata.org/numba-doc/latest/cuda/overview.html', '#cudatoolkit-lookup')
  warnings.warn(errors.NumbaWarning(msg))
[  41 1600 1408]
feature_map_size [1, 200, 176]
remain number of infos: 3769
Generate output labels...
[100.0%][===================>][17.54it/s][03:38>00:00]   
generate label finished(17.20/s). start eval:
avg example to torch time: 0.930 ms
avg prep time: 2.433 ms
avg voxel_feature_extractor time = 0.120 ms
avg middle forward time = 21.836 ms
avg rpn forward time = 21.631 ms
avg predict time = 10.925 ms
/home/ahmed/anaconda3/envs/pointpillars/lib/python3.7/site-packages/numba/core/typed_passes.py:314: NumbaPerformanceWarning: 
The keyword argument 'parallel=True' was specified but no transformation for parallel execution was possible.

To find out why, try turning on parallel diagnostics, see https://numba.pydata.org/numba-doc/latest/user/parallel.html#diagnostics for help.

File "utils/eval.py", line 129:
@numba.jit(nopython=True, parallel=True)
def box3d_overlap_kernel(boxes,
^

  state.func_ir.loc))
Evaluation official
Car AP(Average Precision)@0.70, 0.70, 0.70:
bbox AP:0.00, 0.00, 0.00
bev  AP:0.00, 0.00, 0.00
3d   AP:0.00, 0.00, 0.00
aos  AP:0.00, 0.00, 0.00
Car AP(Average Precision)@0.70, 0.50, 0.50:
bbox AP:0.00, 0.00, 0.00
bev  AP:0.00, 0.00, 0.00
3d   AP:0.00, 0.00, 0.00
aos  AP:0.00, 0.00, 0.00

Evaluation coco
Car coco AP@0.50:0.05:0.95:
bbox AP:0.00, 0.00, 0.00
bev  AP:0.00, 0.00, 0.00
3d   AP:0.00, 0.00, 0.00
aos  AP:0.00, 0.00, 0.00

**Comment by heesungsung** [comment]
Have you solved it? I have the same problem.

**Comment by CuberrChen** [comment]
Have you solved it? I have the same problem.

**Comment by Yacovitch** [comment]
Are you doing an evaluation? If it is, check your model path. It is most likely that you are not feeding the pre-trained model properly into your network.
